src/index.html

```diff
@@ -4,9 +4,400 @@
+    <header class="nav">
+      <div class="nav__container">
+        <a class="nav__logo" href="index.html"><img src="./images/logo.png" alt="Moyo_logo"></a>
+        <nav>
+          <ul class="nav__list">
+            <li class="nav__list-item">
+              <a class="nav__list-link is-active"
+                 href="#apple"
+                 data-qa="nav-hover"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__list-link" href="#samsung">
+                Samsung
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__list-link" href="#smartphones">
+                Smartphones
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__list-link"
+                 href="#laptops & computers"
+                 data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__list-link" href="#gadgets">
+                Gadgets
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__list-link" href="#tablets">
+                Tablets
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__list-link" href="#photo">
+                Photo
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a class="nav__list-link" href="#video">
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="card">
+      <div class="card__container" data-qa="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="imac"
+        >
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br>
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews-text">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-title">
+            Price:
+          </p>
+          <p class="card__price-cost">
+            $2,199
+          </p>
+        </div>
+
+        <div class="card__buy">
+          <a class="card__buy-link"
+             href="#"
+             data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+      <div class="card__container">
```

Comment:
Add the blank line between the cards for better readability
```suggestion
      </div>
      
      <div class="card">
```